### PR TITLE
Signup: update the text in the theme step to show we have more themes to pick from

### PR DESCRIPTION
**Pull request by fditrapani**
This PR updates the copy on the theme step in signup to show that we have more themes to pick from. 

Updated screenshot:
image

## Testing
* Visit http://calypso.localhost:3000/start to start the signup process
* Pick a layout
* See the themes screen.

@markryall @taggon can you please review?

### CODE REVIEW

File: client/signup/steps/theme-selection/index.jsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ class ThemeSelectionStep extends Component {
 		const { translate } = this.props;
 		const headerText = translate( 'Choose a theme.' );
 		const subHeaderText = translate(
-			'No need to overthink it. You can always switch to a different theme later.',
+			'Pick one of our popular themes to get started or choose from hundreds more after you sign up.',
```
Comment (a8ci18n) [inline client/signup/steps/theme-selection/index.jsx]: :information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?

**Comment by taggon** [comment]
LGTM  👍

File: client/signup/steps/theme-selection/index.jsx (hunk 1 above)
Comment (yoavf) [inline client/signup/steps/theme-selection/index.jsx]: +1 - no need for context here. Can be a comment, but I think that's not needed either.

Comment (fditrapani) [inline client/signup/steps/theme-selection/index.jsx]: Sounds good, I'll take it out.

File: client/signup/steps/theme-selection/index.jsx [hunk 2]
```diff
@@ -76,8 +76,7 @@ class ThemeSelectionStep extends Component {
 		const { translate } = this.props;
 		const headerText = translate( 'Choose a theme.' );
 		const subHeaderText = translate(
-			'No need to overthink it. You can always switch to a different theme later.',
-			{ context: 'Themes step subheader in Signup' }
+			'Pick one of our popular themes to get started or choose from hundreds more after you sign up.'
```
Comment (a8ci18n) [inline client/signup/steps/theme-selection/index.jsx]: Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Pick one of our popular themes to get started or choose from hundreds more after you sign up.', { context: 'Themes step subheader in Signup'} )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (fditrapani) [inline client/signup/steps/theme-selection/index.jsx]: 👍

Comment (yoavf) [inline client/signup/steps/theme-selection/index.jsx]: heh :) Safe to ignore of course :)

Comment (fditrapani) [inline client/signup/steps/theme-selection/index.jsx]: No worries. good to know how it works for next time :)

**Comment by designsimply** [comment]
@fditrapani, I noticed the `[Status] String Freeze` label was added to this PR 8 days ago. Does that mean it's on hold waiting for something?

I'm asking because it's the first time I've seen that label used and I was wondering how long to leave PRs with that label before checking in on them (which is something I do as part of my work on the Flow Patrol team that involves triaging and sorting bugs). 🙂

**Comment by fditrapani** [comment]
Thanks for following up @designsimply. The `[Status] String Freeze` is used for strings that are waiting to be translated. It can take a little more thank a week for translations to be completed. We're currently at 94%. Once the translations are complete, I'll get a green light to merge the PR. You can get more info by checking out this post: p4TIVU-5Jq-p2

**Comment by designsimply** [comment]
Perfect, thanks!

**Pull request merged**